### PR TITLE
Support more materialized view syntax

### LINKED ISSUE #4252: Support more materialized view syntax
Consider that future materialized views may contain multiple tables (currently limited to a single table).
 The syntax associated with materialized views should not inherit from the original ROLLUP syntax or be table level. It should be at the database level.

- Create
create materialized view as <select_statement>
- Drop
drop materialized view on database
- Show 
show materialized view on database
- Alter 
show alter materialized view on database 

The problem to be solved is that the materialized views under the same database may have the same name. To solve this problem. The newly created materialized view will perform a uniqueness check, while the original materialized view will display "table_name + mv_name" in the Show Materialized View on Database as the name of the materialized view.

### PR DESCRIPTION
**Pull request by HangyuanLiu**
FIX #4252

- CREATE
Add a judgment of uniqueness when create a materialized view

- DROP
DROP MATERIALIZED VIEW [ IF EXISTS ] [db_name].< mv_name >
 To compatibility with the old rollup logic,  the syntax "DROP MATERIALIZED VIEW [ IF EXISTS ] [db_name].< mv_name > FROM [db].[table]" has been added

- SHOW
SHOW MATERIALIZED VIEW [IN|FROM db_name] 
SHOW ALTER MATERIALIZED VIEW FROM db_name

- CANCEL 
CANCEL ALTER MATERIALIZED VIEW FROM db_name.mv_name

### CODE REVIEW

**Comment by EmmyMiao87** [comment]
`To compatibility with the old rollup logic, the syntax "DROP MATERIALIZED VIEW [ IF EXISTS ] [db_name].< mv_name > FROM [db].[table]" has been added`
Both FROM and ON should be supported.

**Comment by EmmyMiao87** [comment]
The syntax of `SHOW`, `ALTER`, `DROP` should be consistent

File: fe/fe-core/src/main/java/org/apache/doris/analysis/DropMaterializedViewStmt.java [hunk 1]
```diff
@@ -38,38 +43,91 @@
...
     @Override
     public void analyze(Analyzer analyzer) throws UserException {
-        if (Strings.isNullOrEmpty(mvName)) {
-            throw new AnalysisException("The materialized name could not be empty or null.");
+        if (dbTblName != null && !Strings.isNullOrEmpty(dbMvName.getDb())) {
+            throw new AnalysisException("Syntax drop materialized view [mv-name] from db.name mush specify database name explicitly in `from`");
+        }
+        if (dbTblName != null) {
+            if (!Strings.isNullOrEmpty(dbMvName.getDb())) {
+                throw new AnalysisException("If the database appears after the from statement, " +
```
Comment (EmmyMiao87) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/DropMaterializedViewStmt.java:87]: What's the difference between here and above?

File: fe/fe-core/src/main/java/org/apache/doris/analysis/DropMaterializedViewStmt.java [hunk 2]
```diff
@@ -38,38 +43,91 @@
...
+            return dbMvName.getDb();
+        }
     }
 
     @Override
     public void analyze(Analyzer analyzer) throws UserException {
-        if (Strings.isNullOrEmpty(mvName)) {
-            throw new AnalysisException("The materialized name could not be empty or null.");
+        if (dbTblName != null && !Strings.isNullOrEmpty(dbMvName.getDb())) {
+            throw new AnalysisException("Syntax drop materialized view [mv-name] from db.name mush specify database name explicitly in `from`");
```
[the remark this answers is not in the document]
Comment (HangyuanLiu) [reply fe/fe-core/src/main/java/org/apache/doris/analysis/DropMaterializedViewStmt.java:83]: If db set in from , it mush not be set in view name

**Pull request closed without merging**